en/lessons/basics/date-time.md

```diff
@@ -0,0 +1,98 @@
+---
+version: 1.0.0
+title: Date and Time
+---
+
+Working with time in Elixir.
+
+{% include toc.html %}
+
+## Time
+
+Elixir has some modules which work with time. Though it needs to be noted that this functionality is limited to working with UTC timezone.
```

Comment:
Slight improvement:
```suggestion
Elixir has some modules which work with time. Though it needs to be noted that this functionality is limited to working with the UTC timezone only.
```
I would personally join these together even and re-write them slightly:
```suggestion
Elixir has dedicated modules for working with time, though it needs to be noted that this functionality is limited to working with the UTC timezone only.
```

Reply:
Thanks here as well!